docs/content/releases.md

```diff
@@ -6,6 +6,37 @@ doctypes: ["concept"]
 toc: true
 ---
 
+## NGINX Ingress Controller 1.12.1
+
+8 September 2021
+
+CHANGES:
+* Update NGINX App Protect version to 3.5.
+
+HELM CHART:
+* The version of the Helm chart is now 0.10.1.
+
+UPGRADE:
+* For NGINX, use the 1.12.1 image from our DockerHub: `nginx/nginx-ingress:1.12.1`, `nginx/nginx-ingress:1.12.1-alpine` or `nginx/nginx-ingress:1.12.1-ubi`
+* For NGINX Plus, use the 1.12.1 image from the F5 Container Registry - see [the documentation here](https://docs.nginx.com/nginx-ingress-controller/installation/pulling-ingress-controller-image/)
+* Alternatively, you can also build your own image using the 1.12.1 source code.
+* For Helm, use version 0.10.1 of the chart.
+
+## NGINX Ingress Controller 1.12.1
+
+7 September 2021
+
+CHANGES:
+* Update NGINX App Protect version to 3.5.
+
+HELM CHART:
+* The version of the Helm chart is now 0.10.1.
+
+UPGRADE:
+* For NGINX, use the 1.12.1 image from our DockerHub: `nginx/nginx-ingress:1.12.1`, `nginx/nginx-ingress:1.12.1-alpine` or `nginx/nginx-ingress:1.12.1-ubi`
+* For NGINX Plus, use the 1.12.1 image from the F5 Container Registry - see [the documentation here](https://docs.nginx.com/nginx-ingress-controller/installation/pulling-ingress-controller-image/)
+* Alternatively, you can also build your own image using the 1.12.1 source code.
+* For Helm, use version 0.10.1 of the chart.
```

Comment:
Changelog for 1.12.1 twice

Reply:
Good catch thanks! I wonder how that happened, it's fine in https://docs.nginx.com/nginx-ingress-controller/releases/ 🤷🏼